website/content/api-docs/admin-partitions.mdx

```diff
@@ -0,0 +1,258 @@
+  DNS hostname label. If present in the payload it must match what was given
+  in the URL path.
+
+- `Description` `(string: "")` - Free form partition description.
+
+### Sample Payload
+
+```json
+{
+  "Description": "North America West Partition"
+}
+```
+
+### Sample Request
+
+```shell-session
+$ curl -X PUT \
+   -H "X-Consul-Token: 5cdcae6c-0cce-4210-86fe-5dff3b984a6e" \
+   --data @payload.json \
+   http://127.0.0.1:8500/v1/partition/na-west
+```
+
+### SampleResponse
+
+```json
+{
+  "Name": "na-west",
+  "Description": "North America West Partition",
+  "CreateIndex": 55,
+  "ModifyIndex": 60
+}
+```
+
+## Delete a Partition
+
+This endpoint marks a Partition for deletion. Once marked Consul will
+deleted all the associated partitioned data in the background. Only once
+all associated data has been deleted will the Partition actually disappear.
+Until then, further reads can be performed on the partition and a `DeletedAt`
+field will now be populated with the timestamp of when the Partition was
+marked for deletion.
+
+| Method   | Path               | Produces |
+| -------- | ------------------ | -------- |
+| `DELETE` | `/partition/:name` | N/A      |
+
+This endpoint will return no data. Success or failure is indicated by the status
+code returned.
+
+The table below shows this endpoint's support for
+[blocking queries](/api/features/blocking),
+[consistency modes](/api/features/consistency),
+[agent caching](/api/features/caching), and
+[required ACLs](/api#authentication).
+
+| Blocking Queries | Consistency Modes | Agent Caching | ACL Required     |
+| ---------------- | ----------------- | ------------- | ---------------- |
+| `NO`             | `none`            | `none`        | `operator:write` |
+
+### Parameters
+
+- `name` `(string: <required>)` - Specifies the partition to delete. This
+  is required and is specified as part of the URL path.
+
+### Sample Request
+
+```shell-session
+$ curl -X DELETE \
+   -H "X-Consul-Token: b23b3cad-5ea1-4413-919e-c76884b9ad60" \
+   http://127.0.0.1:8500/v1/partition/na-west
+```
+
+### Sample Read Output After Deletion Prior to Removal
+
+```json
+{
+  "Name": "na-west",
+  "Description": "North America West Partition",
+  "DeletedAt": "2021-12-14T23:00:00Z",
+  "CreateIndex": 55,
+  "ModifyIndex": 100
+}
+```
+
+## List all Partitions
+
+This endpoint lists all the Partitions.
+
+| Method | Path          | Produces           |
+| ------ | ------------- | ------------------ |
+| `GET`  | `/partitions` | `application/json` |
+
+The table below shows this endpoint's support for
+[blocking queries](/api/features/blocking),
+[consistency modes](/api/features/consistency),
+[agent caching](/api/features/caching), and
+[required ACLs](/api#authentication).
+
+| Blocking Queries | Consistency Modes | Agent Caching | ACL Required    |
+| ---------------- | ----------------- | ------------- | --------------- |
+| `NO`             | `consistent`      | `none`        | `operator:read` |
```

Comment:
Would like eyes here

Reply:
The consistency mode should be `all`, like with the read endpoint above